src/core_modules/capture-core/components/LockedSelector/LockedSelector.actions.js

```diff
@@ -2,30 +2,34 @@
 import { actionCreator } from '../../actions/actions.utils';
 
 export const lockedSelectorActionTypes = {
-    ORG_UNIT_ID_SET: 'OrgUnitSet',
-    ORG_UNIT_ID_RESET: 'OrgUnitIdReset',
-    PROGRAM_ID_SET: 'ProgramIdSet',
-    PROGRAM_ID_RESET: 'ProgramIdReset',
-    CATEGORY_OPTION_SET: 'CategoryOptionSet',
-    CATEGORY_OPTION_RESET: 'CategoryOptionReset',
-    ALL_CATEGORY_OPTIONS_RESET: 'AllCategoryOptionsReset',
-
-    SELECTIONS_FROM_URL_UPDATE: 'SelectionsFromUrlUpdate',
-    SELECTIONS_FROM_URL_VALID: 'SelectionsFromUrlValid',
-    SELECTIONS_FROM_URL_INVALID: 'SelectionsFromUrlInvalid',
-
-    BASED_ON_URL_ORG_UNIT_SET: 'BasedOnUrlOrgUnitSet',
-    BASED_ON_URL_ORG_UNIT_ERROR_RETRIEVING: 'BasedOnUrlOrgUnitErrorRetrieving',
-    BASED_ON_URL_ORG_UNIT_EMPTY_SET: 'BasedOnUrlOrgUnitEmptySet',
-
-    NEW_REGISTRATION_PAGE_OPEN: 'NewRegistrationPageOpen',
-    SEARCH_PAGE_OPEN: 'SearchPageOpen',
+    ORG_UNIT_ID_SET: 'LockedSelector.OrgUnitSet',
```

Comment:
Hope you dont disslike my change here. It actually helped me a lot while working with complex redux sequences to have an idea where the actions come from. 

Hope this wont be a no go for you

Reply:
No strong opinion here. I've never used the period to namespace the action type before, but lets try it 👍